README.md

```diff
@@ -1121,6 +1122,34 @@ private final SpConfig spConfig = new SpConfigDataBuilder()
         .build();
 ```
 
+## Check end-user consent status for U.S. Multi-State Privacy
+
+Your organization can check an end-user's consent status for a privacy choice by checking the `statuses` object for `usnat` in the `SpConsent` object.
+
+In the following code snippets, replace `{status}` with a status from the table below:
+
+| **Status**            | **Description**                                                                                                                                                                                                                                                              |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sellStatus`          | Checks end-user consent status for _Sale of Personal Information_ privacy choice.                                                                                                                                                                                            |
+| `shareStatus`         | Checks end-user consent status for _Sharing of Personal Information/Targeted Advertising_ privacy choice.                                                                                                                                                                    |
+| `sensitiveDataStatus` | Checks end-user consent status for _Processing of Sensitive Personal Information_ privacy choice.<br><br>Each sensitive data category configured by your organization is collated under this single privacy choice.The end-user either opts into all the categories or none. |
+
+The code snippets will check the configured status and execute the code you have set up for when an end-user is opted-out or opted-into a privacy choice, respectively.
+
+```java
```

Comment:
this should be marked as a `Kotlin` code, `Java` doesn't have a `?` symbol in its disposal

Reply:
Thanks @bohdan-go-wombat I have updated it to kotlin